### PR TITLE
Fix Nova and model tests

### PR DESCRIPTION
**Pull request by kylebarney**

### CODE REVIEW

**Comment by drewroberts** [comment]
Do we need to mock the Cloudinary image size return? Or should we just remove that test?

File: tests/Unit/Models/ImageTest.php [hunk 1]
```diff
@@ -4,12 +4,12 @@
 
 use DrewRoberts\Media\Models\Image;
 use DrewRoberts\Media\Models\Video;
-use DrewRoberts\Media\Tests\Support\Models\User;
 use DrewRoberts\Media\Tests\TestCase;
 use Illuminate\Database\Eloquent\Collection;
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Illuminate\Foundation\Testing\WithFaker;
 use phpmock\phpunit\PHPMock;
```
[the remark this answers is not in the document]
Comment (kylebarney) [reply tests/Unit/Models/ImageTest.php]: So after a bit of digging it turns out that `PHPMock` is an extension of sorts for mocking built-in PHP functions. (Like `getimagesize` in this case.) I've gone ahead and switched this to use `Mockery` (the mock package the `tipoff/test-support` package provides) and its `PHPMock` extension.

**Comment by kylebarney** [comment]
@drewroberts The mock for the image size check is working, so this should be good to go.

File: tests/Unit/Models/ImageTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (drewroberts) [reply tests/Unit/Models/ImageTest.php]: Thank you!

File: composer.json [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
+        "cloudinary/cloudinary_php": "^1.20.0",
+        "silvanite/nova-field-cloudinary": "^1.3.2",
         "spatie/eloquent-sortable": "^3.11.0",
         "tipoff/authorization": "^2.5.1",
         "tipoff/laravel-google-api": "^1.3.1",
         "tipoff/support": "^1.6.0"
     },
     "require-dev": {
-        "php-mock/php-mock-phpunit": "^2.6",
+        "php-mock/php-mock-mockery": "^1.3",
```
Comment (drewroberts) [inline composer.json:29]: Should we add this to tipoff/test-support so it can be used in other packages?

**Review by drewroberts — APPROVED** [review]
Thank you! 🎄

**Pull request merged**